game/src/main/java/org/apollo/game/message/handler/BankMessageHandler.java

```diff
@@ -51,12 +51,12 @@ public BankMessageHandler(World world) {
 
 	@Override
 	public void handle(Player player, ItemActionMessage message) {
-		if (player.getInterfaceSet().contains(BankConstants.BANK_WINDOW_ID)) {
-			if (message.getInterfaceId() == BankConstants.SIDEBAR_INVENTORY_ID) {
+		if (player.getInterfaceSet().contains(BankConstants.WINDOW_ID)) {
+			//if (message.getInterfaceId() == BankConstants.SIDEBAR_INVENTORY_ID) {
```

Comment:
Odd code here. Why is this commented out?

Reply:
Honestly I don't remember. All interfaces have to be redone regardless.

Reply:
@garyttierney It's because the interface doesn't exist anymore. I've removed it entirely now.